codec-http2/src/main/java/io/netty/handler/codec/http2/Http2GoAwaySentEvent.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License, version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.netty.handler.codec.http2;
+
+/**
+ * Notification that a GOAWAY frame has been sent.
```

Comment:
nit: maybe add a `href` to the RFC section to `GOAWAY`

Reply:
Also do we need to clarify that this not actually means that it was successfully written to the wire yet ?

Reply:
done